sources.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"github.com/klauspost/cpuid"
 	"io/ioutil"
 	"os/exec"
 	"path"
-
-	"github.com/klauspost/cpuid"
+	"strconv"
```

Comment:
It seems like the import ordering is messed-up here. Can you please order them? See https://github.com/golang/go/wiki/CodeReviewComments#imports for details.